exo-deploy/terraform/aws/alb/route53.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_route53_record" "external" {
```

Comment:
Security groups are free, so creating one per ALB shouldn't be a problem. That said, you'd still need to authorize traffic from those security groups to the ECS instances, so that would really just move the problem.

One option could be to use the [data source](https://www.terraform.io/docs/providers/aws/d/security_group.html) to find the proper security group within the VPC from inside the service modules. Using something like `coalesce()`, this could be the default, while still leaving the possibility for the user of the module to override if they know better.